sentry/config.example.yml

```diff
@@ -76,6 +76,7 @@ transaction-events.force-disable-internal-project: true
 # GitHub Integration #
 ######################
 
+# github-app.extended-permissions: ['repo']
```

Comment:
I think this is supposed to be `github-login.extended-permissions`. Or are they the same?

Reply:
They are the same on self-hosted (when in `SENTRY_SINGLE_ORGANIZATION` mode). See the hack here: 

https://github.com/getsentry/sentry/pull/21041/files#diff-208a1aca957f5fa6a22fb24ad1141cecR145-R151